apps/studio/components/layouts/StorageLayout/StorageMenu.tsx

```diff
@@ -61,6 +78,34 @@ const StorageMenu = () => {
           >
             New bucket
           </ButtonTooltip>
+
+          <InnerSideBarFilters className="px-0">
+            <InnerSideBarFilterSearchInput
+              name="search-buckets"
+              aria-labelledby="Search buckets"
+              placeholder="Search buckets..."
+              value={searchText}
+              onChange={(e) => {
+                setSearchText(e.target.value.trim())
```

Comment:
Maybe move the `.trim()` inside the `.sort()` call? Doing it here breaks the spacebar when typing into the input lol

Reply:
oh yes! good catch